docs/config.md

```diff
@@ -0,0 +1,37 @@
+# Configuration
+
+The CSI driver instance is configured by a combination of command-line flags and a YAML configuration
+file. You can use the following flags:
+
+- `--config`: path to the YAML configuration file, optional
+- `--nodeid`: the current Kubernetes node identifier, must be the same than the node it's running on
+- `--endpoint`: CSI driver API endpoint for Kubernetes kubelet
+- `--drivername`: CSI driver name to be registered in the cluster
+- `--maxvolumespernode`: maximum amount of volumes per node
+
+During the rollout of the CSI driver it captures the node-id via `env.ValueFrom`
+[directive](./deploy/csi-hostpath-plugin.yaml).
+
+## Configuration File
+
+By default the CSI driver expects the configuration at `/var/run/configmaps/config/config.yaml`, and
+the following attributes:
+
+```yml
+---
+# interval to relist all "Share" object instances
+shareRelistInterval: 10m
+
+# toggles actively watching for resources, when disabled it will only read objects before mount
+refreshResources: true
+
+# list of namespace names ignored
+ignoredNamespaces: []
+```
+
+When the file is not present, the driver assumes default values instead. And, when the configuration
```

Comment:
ok here you go :-) .... you said we'll assume defaults, good.

but you've got that exit up top in main.go if you could not load the config.

And I'm not seeing how in the code it is being achieved yet ... if it is suppose to be the new yaml in the deploy dir, I'm not seeing that as the best way (vs. the code seeding things if the file is missing).

Reply:
You may want to consider [unit-testing](https://github.com/otaviof/csi-driver-shared-resource/blob/04372abf4f3b05cf1e01395d64cf87207f8adf61/pkg/config/manager_test.go#L26-L45) to assert there are defaults, and those defaults can be overwritten by configuration file.

Reply:
OK the unit test did not help with my "how" question, your pointer in your comment https://github.com/openshift/csi-driver-shared-resource/pull/60#discussion_r731093815 did ... so I'm good wrt this.